tests/test_data/elife_poa_e12717.xml

```diff
@@ -1 +1 @@
="par-1"><funding-source><institution-wrap><institution-id institution-id-type="FundRef">http://dx.doi.org/10.13039/100000065</institution-id><institution>National Institute of Neurological Disorders and Stroke</institution></institution-wrap></funding-source><award-id>1R01NS066936</award-id><principal-award-recipient><name><surname>Solecki</surname><given-names>David J</given-names></name></principal-award-recipient></award-group><award-group id="par-2"><funding-source><institution-wrap><institution-id institution-id-type="FundRef">http://dx.doi.org/10.13039/100000912</institution-id><institution>March of Dimes Foundation</institution></institution-wrap></funding-source><award-id>#1-FY12-455</award-id><principal-award-recipient><name><surname>Solecki</surname><given-names>David J</given-names></name></principal-award-recipient></award-group><funding-statement>The funders had no role in study design, data collection and interpretation, or the decision to submit the work for publication.</funding-statement></funding-group></article-meta></front><back><sec id="s1" sec-type="additional-information"><title>Additional information</title><fn-group content-type="competing-interest"><title>Competing interest</title><fn fn-type="conflict" id="conf1"><p>The authors declare that no competing interests exist.</p></fn></fn-group><fn-group content-type="ethics-information"><title>Ethics</title><fn fn-type="other"><p>Animal experimentation: All mouse lines were maintained in standard conditions in accordance with guidelines established and approved by Institutional Animal Care and Use Committee at St. Jude Children's Research Hospital (protocol number = 483).</p></fn></fn-group></sec><sec id="s2" sec-type="supplementary-material"><title>Additional Files</title><sec id="s3" sec-type="datasets"><title>Major datasets</title><p>The following datasets were generated:</p><p><related-object content-type="generated-dataset" id="dataro1" source-id="https://www.ebi.ac.uk/arrayexpress/browse.html" source-id-type="uri"><collab>Shalini Singh</collab>, <collab>David Solecki</collab>, <year>2015</year><x>,</x> <source>E-MTAB-3557</source><x>,</x> <ext-link ext-link-type="uri" xlink:href="https://www.ebi.ac.uk/arrayexpress/browse.html">https://www.ebi.ac.uk/arrayexpress/browse.html</ext-link><x>,</x> <comment>Publicly available at the EBI European Nucleotide Archive (E-MTAB-3557).</comment></related-object><related-object content-type="generated-dataset" id="dataro2" source-id="https://www.ebi.ac.uk/arrayexpress/browse.html" source-id-type="uri"><collab>Rosmaninho</collab>, <collab>Raposo</collab>, <collab>Castro</collab>, <year>2015</year><x>,</x> <source>E-MTAB-3560</source><x>,</x> <ext-link ext-link-type="uri" xlink:href="https://www.ebi.ac.uk/arrayexpress/browse.html">https://www.ebi.ac.uk/arrayexpress/browse.html</ext-link><x>,</x> <comment>Publicly available at the EBI European Nucleotide Archive (E-MTAB-3560).</comment></related-object></p></sec></sec></back></article>
```

Comment:
can this be reformatted too or does it risk to influence the test result?

Reply:
In these cases we produce XML with no additional whitespace in production, and they represent a testing the process. It is possible we could do the tests comparing what the pretty print of the XML library produces, and it should also be an exact match, but the output would not be what we produce in production. My preference would be to not alter them by adding additional whitespace.

Since they are valid XML files, I usually just view in a web browser to see easier to read formatted data, in these sorts of files.